src/engraving/rw/read206/read206.cpp

```diff
@@ -216,29 +216,29 @@ void Read206::readTextStyle206(MStyle* style, XmlReader& e, ReadContext& ctx, st
             }
         } else if (tag == "sizeIsSpatiumDependent" || tag == "spatiumSizeDependent") {
             sizeIsSpatiumDependent = e.readInt();
-        } else if (tag == "frameWidth") {   // obsolete
-            frameType = FrameType::SQUARE;
-            /*frameWidthMM =*/ e.readDouble();
-        } else if (tag == "frameWidthS") {
-            frameType = FrameType::SQUARE;
-            frameWidth = Spatium(e.readDouble());
-        } else if (tag == "frame") {
-            frameType = e.readInt() ? FrameType::SQUARE : FrameType::NO_FRAME;
+        } else if (tag == "borderWidth") {   // obsolete
```

Comment:
Again such `tag` lines should not be changed because they are about what's seen in MuseScore 2 files